sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -342,6 +342,26 @@ class ReturnHelper {
 
 } // anonymous namespace
 
+// SYCL_PI_LEVEL_ZERO_USE_COMPUTE_ENGINE can be set to an integer (>=0) in
```

Comment:
Maybe e can follow the same format as that for copy engine? 'lower_index:upper_index ?

Reply:
I didn't see value in it, but it can be added later if needed.